docs/pages/discordbot/custombot.md

```diff
@@ -0,0 +1,26 @@
+!!! info
+     Let the *Admins* know that you want to run your own bot instead of the Notifiarr bot so we can enable it for your account. *Keep in mind this is a Sub feature.*
```

Comment:
```suggestion
     Let the *Staff* know that you want to run your own bot instead of the Notifiarr bot so we can enable it for your account. *Keep in mind this is a Sub feature.*
```

cascade reword down please.